gapis/api/vulkan/api/physical_device.api

```diff
@@ -47,6 +47,10 @@
   @unused VkPhysicalDeviceProperties         PhysicalDeviceProperties
 }
 
+@internal class PhyDevProperties{
```

Comment:
Can we use the full name here. Since this will show up in the state block, it would be good to keep it sane.